server/config/local.json-dist

```diff
@@ -27,5 +27,9 @@
   "metrics": {
     "sample_rate": 0
   },
-  "iframe_allowed_origins": ["http://127.0.0.1:8080"]
+  "iframe_allowed_origins": ["http://127.0.0.1:8080"],
+  "csp": {
```

Comment:
If we are forcing `enabled: true` in development mode, anything we put in here will be overridden.